packages/suite/src/middlewares/suite/analyticsMiddleware.ts

```diff
@@ -117,6 +117,23 @@ const analytics =
                 }
                 break;
             }
+            case DISCOVERY.COMPLETE:
+                // eslint-disable-next-line no-case-declarations
```

Comment:
maybe you could get rid of this eslint-ignore just by wrapping body of this case into block `{ ...}` ?

Reply:
done and squashed